config.default

```diff
@@ -326,6 +326,15 @@ movingAveragePoints = 12
 # regionIndicesToPlot should be an integer
 regionIndicesToPlot = 5
 
+# Data source to read for comparison.  There are three options
+# 1 - ERS SSTv3b -- Common in CVDP analysis -- 1854 - 2016
+# 2 - ERS SSTv4 -- Updated version of previous -- 1854 - 2016
+# 3 - HADIsst -- Hadley center analysis -- 1870 - 2016
+# Valid Choices -- Uncomment your choice 
+#observationData = /global/homes/l/lvroekel/HADIsst_nino34.nc
+observationData = /global/homes/l/lvroekel/ERS_SSTv3b_nino34.nc
+#observationData = /global/homes/l/lvroekel/ERS_SSTv4_nino34.nc
```

Comment:
Regarding switching the observations off, I think it would be relatively simple but it's not critical.  Again, if the folks running the ACME analysis are happy, let's leave it as is.